src/lib/components/inbox/conversations/__tests__/__snapshots__/conversations-tests.tsx.snap

```diff
@@ -173,7 +187,7 @@ exports[`looks correct when rendered 1`] = `
                       ]
                     }
                   >
-                    2 DAYS
+                    3 DAYS
```

Comment:
@l2succes and @mennenia, this is happening because the `ConversationSnippet` component uses `moment().fromNow()` - the date should definitely be stubbed in that test or we'll have to update our snapshots every day for the rest of eternity 😁.